client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -61,14 +60,18 @@ class VerificationCodeForm extends Component {
 		} );
 	};
 
-	onCodeSubmit = ( event ) => {
+	onSubmitForm = ( event ) => {
 		event.preventDefault();
 
-		const { rememberMe, twoFactorAuthType } = this.props;
+		const { onSuccess, twoFactorAuthType } = this.props;
 		const { twoStepCode } = this.state;
 
-		this.props.loginUserWithTwoFactorVerificationCode( twoStepCode, rememberMe, twoFactorAuthType ).then( () => {
-			this.props.onSuccess();
+		this.props.recordTracksEvent( 'calypso_two_factor_verification_code_submit' );
+
+		this.props.loginUserWithTwoFactorVerificationCode( twoStepCode, twoFactorAuthType ).then( () => {
+			this.props.recordTracksEvent( 'calypso_two_factor_verification_code_success' );
```

Comment:
To save a few characters :).

This makes the call to `loginUserWithTwoFactorVerificationCode()` a little bit easier to read.